boards/shields/dfrobot_can_bus_v2_0/boards/nrf52840_pca10056.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_SPI_3=y
+CONFIG_SPI_3_NRF_SPIM=y
```

Comment:
Looks like some of your files are missing the final newline...

Reply:
Thanks, now fixed.